moveit_configs_utils/moveit_configs_utils/launches.py

```diff
@@ -186,7 +186,7 @@ def generate_warehouse_db_launch(moveit_config):
     return ld
 
 
-def generate_move_group_launch(moveit_config):
+def generate_move_group_launch(moveit_config, load_capabilities=False):
```

Comment:
I would maybe go for something like this:

```python
capabilities = moveit_config.move_group_capabilities
if type(capabilities) == dict:
    capabilities = moveit_config.move_group_capabilities['capabilities']
ld.add_action(
    DeclareLaunchArgument(
        "capabilities",
        default_value=capabilities,
)
```

that would get us the default behaviour if it's not a `dict` (normal case I believe?) and would retrieve the `capabilities` key in case it is a `dict` (MTC case?).

(note: I haven't checked whether this works)

Reply:
It doesn't work. I think `capabilities` is set somewhere to a `dict`

Reply:
Then you could perhaps do something like:

```python
capabilities = moveit_config.move_group_capabilities
if 'capabilities' in moveit_config.move_group_capabilities:
    capabilities = moveit_config.move_group_capabilities['capabilities']
ld.add_action(
    DeclareLaunchArgument(
        "capabilities",
        default_value=capabilities,
)
```

but I haven't read the issue(s) and PR(s) you linked, so perhaps this all doesn't make sense.

Reply:
thanks, this one worked